samples/subsys/ipc/rpmsg_service/Kconfig.sysbuild

```diff
@@ -6,11 +6,5 @@ source "share/sysbuild/Kconfig"
 
 config RPMSG_REMOTE_BOARD
 	string
-	default "nrf5340dk/nrf5340/cpunet" if $(BOARD) = "nrf5340dk"
-	default "bl5340_dvk/nrf5340/cpunet" if $(BOARD) = "bl5340_dvk"
-	default "lpcxpresso54114/lpc54114/m0" if $(BOARD) = "lpcxpresso54114"
-	default "mps2_an521_remote" if $(BOARD) = "mps2_an521"
-	default "v2m_musca_b1_ns" if $(BOARD) = "v2m_musca_b1"
-	default "esp32_devkitc_wroom_appcpu" if $(BOARD) = "esp32_devkitc_wroom"
-	default "esp32_devkitc_wrover_appcpu" if $(BOARD) = "esp32_devkitc_wrover"
-	default "esp32s3_devkitm_appcpu" if $(BOARD) = "esp32s3_devkitm"
```

Comment:
not good to have boards present which doesn't work for this sample, but afaik this used to work before the sysbuild transition here: https://github.com/zephyrproject-rtos/zephyr/pull/65072

@sylvioalves can you take a look at why this doesn't work anymore ?

Reply:
Checking.

Reply:
@nordicjm @tejlmand LGTM. Better to remove this boards while I finish performing necessary changes to use sysbuild here. This might take a few days due to other stuff. Solution used prior HWMv2 does not fit here anymore.